ext/skeleton/config.m4.in

```diff
@@ -75,7 +79,7 @@ if test "$PHP_%EXTNAMECAPS%" != "no"; then
   dnl   -L$%EXTNAMECAPS%_DIR/$PHP_LIBDIR -lm
   dnl ])
   dnl
-  dnl PHP_SUBST(%EXTNAMECAPS%_SHARED_LIBADD)
+  PHP_SUBST(%EXTNAMECAPS%_SHARED_LIBADD)
```

Comment:
```suggestion
  dnl PHP_SUBST(%EXTNAMECAPS%_SHARED_LIBADD)
```

Reply:
Similar "comment" out the rest of the executable code and that's good to go then.

Thank you.